NYPL/research-catalog-indexer
# Activate v8 index, plus extras

Activates RCI on latest QA v8 index:
 - Update qa config and local sam file
 - Update ES client wrapper to parse and use updated authorization method

Other small fixes:
 - Update to-json method to ensure keys are alpha-sorted
 - Fix bug where partner bibs weren't getting buildingLocationIds

 https://newyorkpubliclibrary.atlassian.net/browse/SCC-4310
